episodes/5-minimal-reproducible-code-draft.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Minimal Reproducible Code"
+teaching: 10
+exercises: 2
+---
+
+::: questions
+-   Which part of my code is causing an error message or an incorrect result?
+-   I want to make my code minimal, but where do I even start?
+-   How do I make non-reproducible code reproducible?
+-   How do I tell whether a code snippet is reproducible or not?
+:::
+
+::: objectives
+-   Identify the step that is generating the error
+-   Implement a stepwise approach to make minimal code
+-   Edit a piece of code to make it reproducible
+-   Evaluate whether a piece of code is reproducible as is or not. If not, identify what is missing.
```

Comment:
Again, I feel like this should go before the previous one... (first figure out whether it is reproducible, then fix it to make it so)